### PR TITLE
Regenerate kevm and kwasm regression tests

### LINKED ISSUE #2409: Re-generate regression tests
The KEVM and KWASM regression tests need to be updated; this is blocking multiple tasks.

### PR DESCRIPTION
**Pull request by ttuegel**
Fixes an error about an unbound variable when the `kollect.sh` script fails. This isn't really a critical fix, because it only occurs when the script fails anyway.

Fixes #2409.

---

###### Review checklist

The author performs the actions on the checklist. The reviewer evaluates the work and checks the boxes as they are completed.

-   [ ] **Summary.** Write a summary of the changes. Explain what you did to fix the issue, and why you did it. Present the changes in a logical order. Instead of writing a summary in the pull request, you may push a clean Git history.
-   [ ] **Documentation.** Write documentation for new functions. Update documentation for functions that changed, or complete documentation where it is missing.
-   [ ] **Tests.** Write unit tests for every change. Write the unit tests that were missing before the changes. Include any examples from the reported issue as integration tests.
-   [ ] **Clean up.** The changes are already clean. Clean up anything near the changes that you noticed while working. This does not mean only spatially near the changes, but logically near: any code that interacts with the changes!

### CODE REVIEW

**Comment by emarzion** [comment]
This does fix the issue in that the script is now exiting gracefully.

However, when running `generate.sh` it only completes a single test (`pop1`) before throwing `Did not collect any Kore files!`. Is this an issue with `kollect.sh` or with `generate.sh`, or am I doing something wrong?

File: scripts/kollect.sh [hunk 1]
```diff
@@ -77,7 +77,7 @@ done
 
 echo '"$@"'
 
-if [[ -z "$kollected" ]]
+if [[ -z "${kollected:-}" ]]
 then
     echo >&2 "Did not collect any Kore files!"
     exit 1
```
Comment (ttuegel) [inline scripts/kollect.sh:83]: This is what we do when it fails. We should add an `else` branch where we remove the temporary directory. We will need to add a variable where we save the name of the temporary directory.

File: scripts/kollect.sh [hunk 2]
```diff
@@ -77,8 +49,10 @@ done
...
 fi
+
+#TODO Mircea: delete temporary folder at the end?
```
Comment (ttuegel) [inline scripts/kollect.sh:58]: TODO?

Comment (MirceaS) [inline scripts/kollect.sh:58]:
```suggestion
```
We agreed to no longer remove the run folder at the end of kollection

**Review by MirceaS — APPROVED** [review]

**Pull request merged**